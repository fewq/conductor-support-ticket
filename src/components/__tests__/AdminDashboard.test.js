import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import AdminDashboard from '../admin/Dashboard';

// more test will be added when there is more frontend development here

test('Renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<AdminDashboard />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('snapshot testing', () => {
  const tree = renderer
    .create(<AdminDashboard />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

