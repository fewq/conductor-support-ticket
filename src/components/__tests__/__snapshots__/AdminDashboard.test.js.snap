// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing 1`] = `
<div>
  <div
    className="container-fluid"
  >
    <nav
      className="navbar navbar-expand-lg navbar-dark bg-dark"
    >
      <a
        className="navbar-brand"
        href="https://beta.acnapi.io/#!/"
        target="_blank"
      >
        <img
          alt="ACNAPI"
          height="30"
          src="acnapi.png"
          width="100"
        />
      </a>
      <a
        className="navbar-brand"
        href="/dashboard"
        onClick={[Function]}
      >
        Ticket Dashboard
      </a>
      <div
        className="collpase navbar-collapse"
      >
        <ul
          className="navbar-nav mr-auto"
        >
          <li
            className="navbar-item"
          >
            <a
              className="nav-link"
              href="/dashboard"
              onClick={[Function]}
            >
              Table View
            </a>
          </li>
          <li
            className="navbar-item"
          >
            <a
              className="nav-link"
              href="/kanban"
              onClick={[Function]}
            >
              Kanban
            </a>
          </li>
        </ul>
      </div>
    </nav>
    <br />
  </div>
</div>
`;
